supertokens_python/recipe/multitenancy/recipe_implementation.py

```diff
@@ -61,13 +69,25 @@ async def delete_tenant(
     ) -> DeleteTenantOkResult:
         raise NotImplementedError
 
-    async def get_tenant_config(
+    async def get_tenant(
         self, tenant_id: Optional[str], user_context: Dict[str, Any]
-    ) -> TenantConfigOkResult:
-        return TenantConfigOkResult(
+    ) -> GetTenantOkResult:
+        _ = self.querier.send_get_request(
+            NormalisedURLPath(
+                f"{tenant_id or DEFAULT_TENANT_ID}/recipe/multitenancy/tenant"
```

Comment:
what if the value of tenant_id is `""`, then will `tenant_id or DEFAULT_TENANT_ID` result in `""` or `"public"`?

Reply:
It will become `"public"` on using `or`